radicle-seed/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "radicle-seed"
+version = "0.1.0"
+authors = ["The Radicle Team <dev@radicle.xyz>"]
+edition = "2018"
+license = "GPL-3.0-or-later"
+
+[dependencies]
+librad = { path = "../librad" }
+async-trait = "0.1"
+thiserror = "1.0"
+tokio = "0.2"
+futures = "0.3"
+argh = "0.1.3"
+log = { version = "0.4", features = ["std"] }
```

Comment:
Should you not remove the `log` dependency now?

Reply:
Done!